modules/localgov_directories_or/localgov_directories_or.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'LocalGov Directories: Open Referral'
+type: module
+description: 'Adds mappings for Facets for Open Referral.'
+package: LocalGov Drupal
+core: 8.x
+core_version_requirement: ^8 || ^9
+dependencies:
+  - localgov_directories:localgov_directories
+  - localgov_openreferral:localgov_openreferral
```

Comment:
Think most of the comments above make sense, shall get to them when looking at this again next week.

On this particular one however:-
 * LocalGov Open Referral will in time have a packagist entry etc. Hadn't got that far yet.
 * LocalGov Directories Organisation is just a Directory type. It can be used as is without Open Referral should you want. It's not even required if you want Directories to output Open Referral. Something mapped to Open Referral Organisation type is required and this module offers it to you without having to do anything custom.
 * LocalGov Open Referral has no code about Directories at all. In theory you can use it with other content types.

So a suggest in the LocalGov Directories composer.json makes sense I think.

Reply:
Gone with the suggests.